db/odm/event.odm.js

```diff
@@ -9,6 +9,7 @@ import Group from 'db/models/Group';
 async function findByUserId(userId) {
   const listEvents = await Group.find(
     { users: userId },
+    { _id: 0 },
```

Comment:
Why adding `{ _id: 0 }` here?

Reply:
As it is Group _id, that is unnecessary

Reply:
https://github.com/EasyEventV2/ee2_api/pull/5/commits/69f0baf704721b8cf494f97228d6737f8742fd87